### ISSUE TITLE
Hello CIOD authors,I was trying to run your code on VOC but couldnot success. With the deafult settings I have some problems. Could you help me figure out the problem? Thanks!

### ISSUE THREAD

**Issue by CHyan28** [jinyu121/CIOD#7]
1. I was trying to run your code on VOC. When the group>1, I appear as follows:
发生异常: IndexError
too many indices for tensor of dimension 1
  File "/home/wangmingkun/faster/faster-rcnn.pytorch/lib/roi_data_layer/roibatchLoader_incre.py", line 192, in __getitem__
    not_keep = (gt_boxes[:, 0] == gt_boxes[:, 2]) | (gt_boxes[:, 1] == gt_boxes[:, 3])
  File "/home/wangmingkun/faster/faster-rcnn.pytorch/train_incre.py", line 388, in <module>
    data = next(data_iter)
2.  When the group=1, lr = 0.001, batch = 4,  the training loss is Nan.
10022 roidb entries                                                                                                                                                                          
[S1000-0.001 G0] lr: 0.001, loss: 23.03, fg/bg=(43/469)                                                                                                                                      
                        rpn_cls: 0.6497, rpn_box: 0.9001, rcnn_cls: 20.94, rcnn_box 0.5414                                                                                                   
[S1000-0.001 G0] lr: 0.001, loss: nan, fg/bg=(512/0)                                                                                                                                         
                        rpn_cls: nan, rpn_box: nan, rcnn_cls: nan, rcnn_box nan

**Comment by chenfangchenf** [comment]
did you solve this problem？

**Comment by CHyan28** [comment]
Thank you for your reply, I have solved it.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jinyu121/CIOD"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月25日(星期三) 下午3:50
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jinyu121/CIOD] Hello CIOD authors,I was trying to run your code on VOC but couldnot success. With the deafult settings I have some problems. Could you help me figure out the problem? Thanks! (Issue #7)





 
did you solve this problem？
